### PR TITLE
[backend/frontend] Command Details in execution traces (#1232)

### PR DESCRIPTION
**Pull request by damgouj**
### Proposed changes

* Display initial command line in execution details tab
* Update the Front to add the new section "Command lines"
* Update the Back/Database to add command lines for each inject
* Add Integration test for AtomicTestingApi

### Related issues

* https://github.com/OpenBAS-Platform/openbas/issues/1232 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [x] I wrote test cases for the relevant uses case
- [x] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Review by johanah29 — APPROVED** [review]

**Comment by johanah29** [comment]
Should rename your migration fille to 3.37 as 3.36 was merged on master

File: openbas-front/src/components/ThemeDark.ts [hunk 1]
```diff
@@ -175,10 +175,6 @@ const ThemeDark = (
             color: '#ffffff !important',
             background: `${accent || '#01478d'} !important`,
           },
-          'pre.light': {
-            fontFamily: 'Consolas, monaco, monospace',
-            background: `${nav || '#071a2e'} !important`,
-          },
```
Comment (guillaumejparis) [inline openbas-front/src/components/ThemeDark.ts:181]: Not used anymore ?

Comment (damgouj) [reply openbas-front/src/components/ThemeDark.ts:181]: I couldn't find it in the code and there are two files to manage the themes (there is a pre "light" in ThemeLight.ts)

**Comment by damgouj** [comment]
> Seems weird to display commands lines without at least which platform they are for image

Seen with JB : It's just to inform the user, we keep it like that and we'll see if there are remarks in the future.

**Pull request merged**